### ISSUE TITLE
Refactoring: implement high-level example for one new model (e.g. SwAV)

### ISSUE THREAD

**Issue by MalteEbner** [lightly-ai/lightly#492]
Implement a high-level example of a new model (e.g. SwaV) This model should also include the loss function and how to put all together. E.g. write the corresponding pytorch lightning module. For inspiration how it looks like, look at https://github.com/lightly-ai/lightly/blob/483-SSL-model-refactoring-remove-models%2C-provide-full-examples-instead-me/lightly/models/examples/moco_system.py

## Dependencies
- https://github.com/lightly-ai/lightly/issues/12
